solr/solrj-zookeeper/src/java/org/apache/solr/common/cloud/ZkStateReader.java

```diff
@@ -207,6 +207,10 @@ public static ZkStateReader from(CloudSolrClient solrClient) {
     }
   }
 
+  protected ExecutorService getNotifications(){
```

Comment:
It doesn't return notifications, it returns an Executor to deliver asynchronous notifications on.  I suggest the name getWatcherExecutor or getNotificationExecutor.  Maybe the "Watcher" name because the thread pool is named this, thus that's how this executor is visible.  Anyway, no big deal.

BTW I can see you are forgetting to run "tidy".